flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/read/HiveTableInputFormat.java

```diff
@@ -345,13 +345,10 @@ public InputSplitAssigner getInputSplitAssigner(HiveTableInputSplit[] inputSplit
 
     public int getNumFiles() throws IOException {
```

Comment:
It seems this method is no longer needed. We can just remove it.

Reply:
Got it.